gradle.properties

```diff
@@ -1,4 +1,5 @@
 org.gradle.parallel=true
+org.gradle.caching=true
```

Comment:
what does this do ?

Reply:
In theory it helps caching also when your gralde daemon is not yet started, however, I must say I don't see much difference. So I will remove for now.